packages/controllers/src/permissions/PermissionController.ts

```diff
@@ -218,6 +218,14 @@ export type RequestPermissions = {
   handler: GenericPermissionController['requestPermissions'];
 };
 
+/**
+ * Removes for each origin removes given permissions
+ */
+export type RevokePermissions = {
```

Comment:
This should be added in https://github.com/MetaMask/controllers since this version will be deleted as soon as the new controllers version is released.

Reply:
@rekmarks is planning to release the new version today, so you might be able to get these changes in before then

Reply:
Done https://github.com/MetaMask/controllers/pull/708